pandora-client-web/src/components/wardrobe/modules/wardrobeModuleLockSlot.tsx

```diff
@@ -217,6 +218,26 @@ function WardrobeLockSlotLocked({ target, item, moduleName, lock }: Omit<Wardrob
 			</Row>
 		);
 	}, [lock, now]);
+	const timeLeft = useMemo(() => {
+		const lockedData = lock.lockLogic.lockData.locked;
+		Assert(lockedData != null);
+
+		const { time, timerMinutes } = lockedData;
+		if (timerMinutes == null)
+			return null;
+
+		return time + (timerMinutes * 60_000) - now;
+	}, [lock, now]);
+	const timerText = useMemo(() => {
+		if (timeLeft == null || timeLeft <= 0)
+			return 'Can be unlocked now';
+
+		return (
+			<>
+				Can be unlocked in { FormatTimeInterval(timeLeft) }
```

Comment:
[suggestion, optional] Just making it slightly nicer:
```suggestion
				Remaining time: { FormatTimeInterval(timeLeft) }
```